packages/fuels-programs/src/logs.rs

```diff
@@ -62,23 +63,81 @@ pub struct LogDecoder {
     pub log_formatters: HashMap<LogId, LogFormatter>,
 }
 
+#[derive(Debug)]
+pub struct LogResult {
+    pub results: Vec<Result<String>>,
+}
+
+impl LogResult {
+    pub fn filter_succeeded(&self) -> Vec<&str> {
```

Comment:
```suggestion
    pub fn successful_logs(&self) -> Vec<&str> {
```

It's a simple getter so I think you can use this convention https://rust-lang.github.io/api-guidelines/naming.html#c-getter

Reply:
It's no ordinary getter. Values from `results: Vec<Result<String>>` are filtered. `log_result.filter_succeeded()` is a more accurate description of what it does.